modules/openapi-generator/src/main/resources/csharp-netcore/ApiClient.mustache

```diff
@@ -459,10 +517,10 @@ namespace {{packageName}}.Client
         /// <param name="configuration">A per-request configuration object. It is assumed that any merge with
         /// GlobalConfiguration has been done before calling this method.</param>
         /// <returns>A Task containing ApiResponse</returns>
-        public async Task<ApiResponse<T>> GetAsync<T>(string path, RequestOptions options, IReadableConfiguration configuration = null)
+        public Task<ApiResponse<T>> GetAsync<T>(string path, RequestOptions options, IReadableConfiguration configuration = null)
```

Comment:
Looks like this is a breaking change and will need to target 4.3.x

Reply:
This is not a breaking change. C# async keyword has no effect on method signatures.

Reply:
Awesome. Thanks for clarifying this!